config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -28,6 +29,13 @@ periodics:
           popd
 
           depstat stats -m "k8s.io/kubernetes$(ls staging/src/k8s.io | awk '{printf ",k8s.io/" $0}')" --json | tee "${WORKDIR}/stats-base.json";
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 2
+            memory: 4Gi
```

Comment:
I don't access to metrics related to those jobs so I start with something minimal.